app/code/Magento/InventoryBundle/Plugin/InventoryCatalog/Observer/DisableProcessSourceItemsObserverPlugin.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryBundle\Plugin\InventoryCatalog\Observer;
+
+use Magento\Bundle\Model\Product\Type;
+use Magento\Catalog\Api\Data\ProductInterface;
+use Magento\InventoryCatalog\Observer\ProcessSourceItemsObserver;
+use Magento\Framework\Event\Observer;
+
+/**
+ * Class disable inventory for bundle products
+ */
+class DisableProcessSourceItemsObserverPlugin
```

Comment:
You need to disable observer via events configuration

We don't allow use plugins for plugins, observers, data objects
Generally we think about mark these class as `final` for avoiding pluginization of these classes

Reply:
Ok, get it.